cliquet/authorization.py

```diff
@@ -19,6 +19,9 @@ class AuthorizationPolicy(object):
     get_bound_permissions = None
 
     def permits(self, context, principals, permission):
+        # User id prefixed with authentication policy type.
+        principals.insert(0, context.prefixed_userid)
```

Comment:
We don't need to add it in first position, also we should remove the ``authenticated_userid`` from the principals